src/common/reconciler.go

```diff
@@ -175,6 +147,7 @@ func (r *ServiceReconciler) handleService(registration opslevel_jq_parser.Servic
 	return service, nil
 }
 
+// TODO: update this to more closely follow the logic of updateService()
```

Comment:
I don't think we actually want to do this.  They are different flows logically with different outcomes and different starting states.

Reply:
removed comment